TestApp/Scripts/experience.js

```diff
@@ -68,10 +164,35 @@ CreateBoxAsync().then(function () {
 //BABYLON.SceneLoader.AppendAsync("https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/CesiumMan/glTF/CesiumMan.gltf").then(function () {
     BABYLON.Tools.Log("Loaded");
 
-    scene.createDefaultCamera(true);
-    scene.activeCamera.alpha += Math.PI;
+    if (viewports) {
+        // camera positions set for sponza
+        var camera1 = new BABYLON.FreeCamera("camera1", new BABYLON.Vector3(3, 2, -6), scene);
```

Comment:
how do you switch cameras?

Reply:
You can change the active camera this way:
scene.activeCamera = ...

Reply:
Right, but should we add this code to experience.js if there is no way to change the cameras?

Reply:
Until we have the validation tests, do we keep JS script that helps developing/testing features?  If so, in experience.js or another file?

Reply:
We sort of already have the validation tests in a separate folder: https://github.com/BabylonJS/BabylonNative/tree/master/TestApp/ValidationTests

Reply:
Maybe create a new file in that folder.